clients/src/main/java/org/apache/kafka/common/protocol/Protocol.java

```diff
@@ -993,9 +993,9 @@
 
     /* Transactions API */
     public static final Schema INIT_PRODUCER_ID_REQUEST_V0 = new Schema(
-            new Field("appid",
+            new Field("txnid",
```

Comment:
I think for doc strings spelling it out is totally worth it. In general, I think transactionalID is not too long to spell out in most cases, except maybe an a variable name at  very local level where it is repeated very frequently.